### PR TITLE
fix: bump dependencies and remove ssl option

### PR DESCRIPTION
**Pull request by nicolethoen**
Closes https://github.com/patternfly/patternfly-react/issues/8994

### CODE REVIEW

**Review by wise-king-sullyman — COMMENTED** [review]
LGTM other than the class update that looks like it's needed for the failing integration test.

Also I believe that if we want to we could open the node versioning back up now that we can get away from the legacySSL usage.

**Review by wise-king-sullyman — APPROVED** [review]

File: packages/react-core/src/components/DataList/DataListItem.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ export class DataListItem extends React.Component<DataListItemProps> {
               className={css(
                 styles.dataListItem,
                 isExpanded && styles.modifiers.expanded,
-                isSelectable && styles.modifiers.selectable,
```
Comment (tlabaj) [inline packages/react-core/src/components/DataList/DataListItem.tsx:85]: Does this correlate to an issue?  If so is there still additional work needed?

Comment (nicolethoen) [reply packages/react-core/src/components/DataList/DataListItem.tsx:85]: This does not correlate with an issue - the core issue was marked as needing no react follow up: https://github.com/patternfly/patternfly/issues/5347
@thatblindgeye do you know if there is other follow up work needed for this update?

**Review by tlabaj — COMMENTED** [review]
LGTM

**Review by tlabaj — APPROVED** [review]

**Pull request merged**